tools/slims.py

```diff
@@ -167,23 +135,22 @@ def translate_slims_info(record) -> dict:
     }
     return master
 
-def samples_from_sec_analysis(primary_key: int) -> dict:
+def slims_records_from_sec_analysis(primary_key: int) -> list:
     """
-    Fetch all records from slims given a secondary analysis tag
+    Fetch all DNA objects from slims given a secondary analysis key
 
     :param primary_key: Primary key integer of secondary analysis
-    :return: Dictionary of sample names and their corresponding slims records
+    :return: List with all DNA objects marked with the secondary analysis key
     """
-
-    records = slims.fetch("Content", equals("cntn_cstm_secondaryAnalysis", primary_key))
-
-    samples = {}
+    return_list = []
```

Comment:
`return_list` feels a bit unclear. Maybe `record_ids`?

Reply:
All this code is also being moved over to the slims_link repo. This will be solved when migrating to using the code there. I'll leave it here for now.